drivers/sensor/current_amp/current_amp.c

```diff
@@ -28,7 +28,7 @@ static int fetch(const struct device *dev, enum sensor_channel chan)
 		return -ENOTSUP;
 	}
 
-	ret = adc_read(config->port.dev, &data->sequence);
+	ret = adc_read_dt(config->port.dev, &data->sequence);
```

Comment:
Hopefully CI catches this one?

```suggestion
	ret = adc_read_dt(config->port, &data->sequence);
```

Reply:
Fixed 👍